extensions/sql-migration/src/dialog/migrationCutover/confirmCutoverDialog.ts

```diff
@@ -0,0 +1,133 @@
+					'font-size': '10px',
+					'margin': '5px 0px 10px 0px'
+				}
+			}).component();
+
+			const separator = this._view.modelBuilder.separator().withProps({ width: '800px' }).component();
+
+			let infoDisplay = 'none';
+			if (this.migrationCutoverModel._migration.targetManagedInstance.id.toLocaleLowerCase().includes('managedinstances')
+				&& (<any>this.migrationCutoverModel._migration.targetManagedInstance)?.sku?.tier.toLowerCase().includes('BusinessCritical')) {
+				infoDisplay = 'inline';
+			}
+
+			const businessCriticalinfoBox = this._view.modelBuilder.infoBox().withProps({
+				text: constants.BUSINESS_CRITICAL_INFO,
+				style: 'information',
+				CSSStyles: {
+					'font-size': '13px',
+					'display': infoDisplay
+				}
+			}).component();
+
+			const helpMainText = this._view.modelBuilder.text().withProps({
+				value: constants.CUTOVER_HELP_MAIN,
+				CSSStyles: {
+					'font-size': '13px',
+				}
+			}).component();
+
+			const helpStepsText = this._view.modelBuilder.text().withProps({
+				value: `${constants.CUTOVER_HELP_STEP1}
+				${constants.CUTOVER_HELP_STEP2}
+				${constants.CUTOVER_HELP_STEP3}`,
+				CSSStyles: {
+					'font-size': '13px',
+				}
+			}).component();
+
+			const pendingBackupCount = this.migrationCutoverModel.migrationStatus.properties.migrationStatusDetails?.activeBackupSets.filter(f => f.listOfBackupFiles[0].status !== 'Restored' && f.listOfBackupFiles[0].status !== 'Ignored').length;
+			const pendingText = this._view.modelBuilder.text().withProps({
+				CSSStyles: {
+					'font-size': '13px',
+					'font-weight': 'bold'
+				},
+				value: constants.PENDING_BACKUPS(pendingBackupCount!)
+			}).component();
+
+			const confirmCheckbox = this._view.modelBuilder.checkBox().withProps({
+				CSSStyles: {
+					'font-size': '13px',
+				},
+				label: constants.CONFIRM_CUTOVER_CHECKBOX,
+			}).component();
+
+			confirmCheckbox.onChanged(e => {
+				this._dialogObject.okButton.enabled = e;
+			});
+
+
+			const container = this._view.modelBuilder.flexContainer().withLayout({
+				flexFlow: 'column'
+			}).withItems([
+				completeCutoverText,
+				sourceDatabaseText,
+				separator,
+				businessCriticalinfoBox,
+				helpMainText,
+				helpStepsText,
+				pendingText,
+				confirmCheckbox
+			]).component();
+
+
+			this._dialogObject.okButton.enabled = false;
+			this._dialogObject.okButton.label = constants.COMPLETE_CUTOVER;
+			this._dialogObject.okButton.onClick((e) => {
+				this.migrationCutoverModel.startCutover();
+				vscode.window.showInformationMessage(constants.CUTOVER_IN_PROGRESS(this.migrationCutoverModel._migration.migrationContext.properties.sourceDatabaseName));
+			});
+
+			const formBuilder = view.modelBuilder.formContainer().withFormItems(
+				[
+					{
+						component: container
+					}
+				],
+				{
+					horizontal: false
+				}
+			);
+			const form = formBuilder.withLayout({ width: '100%' }).component();
+			return view.initializeModel(form).then((value) => {
```

Comment:
The .then seems unnecessary - you could just await the call instead of returning it directly.